aci/convert/volume.go

```diff
@@ -139,7 +139,7 @@ type volumeInput struct {
 
 // parse takes a candidate string and creates a volumeInput
 // Candidates take the form of <source>[:<target>][:<permissions>]
-// Source is of the form `<storage account>/<fileshare>`
+// Container is of the form `<storage account>/<fileshare>`
```

Comment:
a find/replace with too large scope ? (and I suppose many more instances following)

Reply:
ouch :'(